migrations/900000000000023_add_vendor_and_provider.up.sql

```diff
@@ -0,0 +1,33 @@
+-- SQL migration to add support for vendor and provider constructs
+BEGIN;
+
+-- Table mergestat.vendor contains a list of all third-party vendors that mergestat supports.
+-- It is almost always read-only for users, and data can only be inserted in this table using
+-- a migration.
+CREATE TABLE mergestat.vendors (
+    name TEXT PRIMARY KEY,     -- the enumerated name of the vendor; try to keep it lower-case
+    displayName TEXT NOT NULL, -- the name shown on the user interface
```

Comment:
```suggestion
    display_name TEXT NOT NULL, -- the name shown on the user interface
```

to be consistent with our other column names

Reply:
I've made the change in #808 (see [this commit](https://github.com/mergestat/mergestat/pull/808/commits/3d6c3cbb1da7f05a43afe8777c03110bc69ac001))